waspc/data/Generator/templates/server/src/auth/providers/oauth/env.ts

```diff
@@ -0,0 +1,21 @@
+import { type ProviderConfig } from "wasp/auth/providers/types";
+
+export function ensureEnvVarsForProvider<EnvVars extends string>(
+  envVarNames: EnvVars[],
```

Comment:
The `EnvVars` should be a singular name (i.e., `EnvVar`). Also, a better name is `EnvVarName`.